config/default.yaml

```diff
@@ -65,9 +66,13 @@ wallet:
   passcode: BWVdmGdn8MeCWJxtcfM77eTMMxUELYL9 # TODO: remove
 network:
   contracts_path: contracts
-  web3: "http://ynet.point.space:44444"
+  web3:
+    ynet: "http://ynet.point.space:44444"
+    rinkeby: "https://rinkeby.infura.io/v3/400b1ad4183b442dbd9b0b5744b327aa"
```

Comment:
@jensendarren @amherag This is my infura account ID, I've just created it. I won't mind it, but this is free account with some rate limits, so this can be a problem if we all start using it.